.circleci/config.yml

```diff
@@ -125,8 +125,8 @@ commands:
       - checkout
       - restore_cache:
           keys:
-            - v${CACHE_VERSION}-dependencies-{{ checksum "package.json" }}
-            - v${CACHE_VERSION}-dependencies-
+            - v${CACHE_VERSION}-{{ arch }}-dependencies-{{ checksum "package.json" }}
+            - v${CACHE_VERSION}-{{ arch }}-dependencies-
```

Comment:
are these changes relevant to this PR?

Reply:
Sort of. They're an addendum to what @limdauto did yesterday.

Reply:
Yea this is needed to separate `linux` cache from `windows` cache in CI.